src/main/java/com/ibm/watson/modelmesh/ModelMesh.java

```diff
@@ -1292,6 +1292,14 @@ boolean isLeader() {
         return le != null && le.isLeader();
     }
 
+    @Override
+    protected boolean isLive() {
```

Comment:
where is this used?

Reply:
And k8s uses livenes probe to detect containers in broken state
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command

Reply:
I meant, where in the code it is referenced.
It is coming from here https://github.com/IBM/litelinks-core/blob/main/src/main/java/com/ibm/watson/litelinks/server/ThriftService.java#L113

Uris: https://github.com/IBM/litelinks-core/blob/b281a155f37b3d419fa4135e09009607f668dc3f/src/main/java/com/ibm/watson/litelinks/server/ProbeHttpServer.java#L56C15-L56C66